build.sbt

```diff
@@ -37,11 +39,16 @@ bintrayOrganization := Some("localytics")
 bintrayPackageLabels := Seq("localytics", "sbt", "aws", "sqs", "elasticmq", "test", "testing")
 
 // Error on conflicting dependencies
-// http://www.scala-sbt.org/0.13/docs/Library-Management.html#Conflict+Management
+// http://www.scala-sbt.org/1.x/docs/Library-Management.html#Conflict+Management
 conflictManager := ConflictManager.strict
 
 // Error on circular dependencies
-// http://www.scala-sbt.org/0.13/docs/sbt-0.13-Tech-Previews.html#Circular+dependency
+// http://www.scala-sbt.org/1.x/docs/sbt-0.13-Tech-Previews.html#Circular+dependency
 updateOptions := updateOptions.value.withCircularDependencyLevel(CircularDependencyLevel.Error)
 
-libraryDependencies += "org.scalatest" %% "scalatest" % "2.2.6" % "test"
+libraryDependencies +=
+  ( "org.scalatest" %% "scalatest" % "3.0.1" % "test"
```

Comment:
It looks like scalatest `3.0.4` is out now. It might be worth testing to see if we can get things building with that version without the excludes here.

Reply:
@jsolmon updating to 3.0.4 is fine so we might as well bump it. It doesn't fix the exclude problem though. Once 3.1.x is release though it should be resolved: https://github.com/scalatest/scalatest/blob/2f49a467420e95c5814f196b892ad1d7d4eb57d7/project/scalatest.scala#L172-L183